mjcf_to_sdformat/tests/test_add_mjcf_worldbody_to_sdf.py

```diff
@@ -0,0 +1,135 @@
+# Copyright (C) 2022 Open Source Robotics Foundation
+
+# Licensed under the Apache License, version 2.0 (the "License")
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#       http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from numpy.testing import assert_allclose
+from dm_control import mjcf
+import os
+
+import sdformat as sdf
+
+from mjcf_to_sdformat.converters.world import mjcf_worldbody_to_sdf
+import sdformat_mjcf_utils.sdf_utils as su
+
+
+class ModelTest(unittest.TestCase):
+
+    def test_add_model(self):
+        mjcf_model = mjcf.from_path(os.path.join(
+                                    os.path.dirname(os.path.abspath(__file__)),
+                                    "resources/test_mujoco.xml"))
```

Comment:
Once https://github.com/gazebosim/gz-mujoco/pull/6 is merged, let's change this to
```suggestion
        mjcf_model = mjcf.from_path(str(TEST_RESOURCES_DIR / "test_mujoco.xml")))
```
after importing `TEST_RESOURCES_DIR`.

Reply:
https://github.com/gazebosim/gz-mujoco/pull/26/commits/689f9ef1fe9c14ae38252aab98d0fe140c89b367